### ISSUE TITLE
postcss.parse issue with resolving props

### ISSUE THREAD

**Issue by hcomnetworkers** [postcss/postcss#1226]
First up, I'm sorry if that is a duplicated issue, i could not find this issue in the list.

We may have found a bug in the parser, with the following code:
```
const root = postcss.parse(`.foo {
    foobar
    color: $bar;
}`);
```

The result of root is a rule with a declaration and the prop `foobar` instead of `color`.
So using it with `postcssJs.objectify(root);` would result in this:

```
{
  .foo {
    foobar: "$bar";
  }
}
```

which is wrong.

**Comment by ai** [comment]
Throwing `CssSyntaxError` will be the best behavior on this invalid CSS input?

**Comment by hcomnetworkers** [comment]
That would be perfect.

**Comment by ai** [comment]
Done b6bcd8c

**Comment by ai** [comment]
Released in 7.0.12
